api/src/models/master/permitAmendment.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+module.exports = require('../../utils/model-schema-generator')(
+  'PermitAmendment',
+  {
+    _schemaName: { type: String, default: 'PermitAmendment', index: true },
+    _sourceRefId:  { type: String, default: null, index: true },
+
+    read: [{ type: String, trim: true, default: 'sysadmin' }],
+    write: [{ type: String, trim: true, default: 'sysadmin' }],
+
+    _flavourRecords: [{ type: 'ObjectId', default: [], index: true }],
+
+    statusCode: { type: String, default: '' },
+    typeCode: { type: String, default: '' },
+    receivedDate: { type: Date, default: null },
+    issueDate: { type: Date, default: null },
+    authorizedEndDate: { type: Date, default: null },
+    description: { type: String, default: '' },
+    documents: [{ type: 'ObjectId', default: [], index: true }],
+
+    dateAdded: { type: Date, default: Date.now() },
+    dateUpdated: { type: Date, default: null },
+    datePublished: { type: Date, default: null },
+
+    addedBy: { type: String, default: '' },
+    updatedBy: { type: String, default: '' },
+    publishedBy: { type: String, default: '' },
```

Comment:
Could add `sourceDateAdded` and `sourceDateUpdated` to be consistent with the other types.

Also, probably need to add `isBcmiPublished` ?  This would be in alignment with the other types, though I'm not sure how collections influences this, if it does.

Reply:
Added `isBcmiPublished` to permit and permit amendments. I don't capture the source date information as none of that is returned by the Core API.

Reply:
> I don't capture the source date information as none of that is returned by the Core API.

Yeah, fair enough then

Reply:
If you add it here, `isBcmiPublished` needs to be done for all other master records in order to facilitate search on the admin page.